### DISCUSSION TITLE
Heading 5

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#321] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 5 component" src="https://github.com/user-attachments/assets/1a877574-8c89-45b7-8ce7-40e5d8263360">

## Naam

Heading 5

## Doel

Koptekst die in de koppenstructuur op het vijfde niveau staat.

## Ook bekend als

Heading, Titel

## Gerelateerde componenten

[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 1](https://github.com/orgs/nl-design-system/discussions/317), [Heading 2](https://github.com/orgs/nl-design-system/discussions/318), [Heading 3](https://github.com/orgs/nl-design-system/discussions/319), [Heading 4](https://github.com/orgs/nl-design-system/discussions/320), [Heading 6](https://github.com/orgs/nl-design-system/discussions/322), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 5 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+5)
